src/js/providers/html5.js

```diff
@@ -614,7 +615,7 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
     };
 
     this.init = function(item) {
-        _this.retries = 0;
```

Comment:
we'll want to keep this on init, see the original comment here https://github.com/jwplayer/jwplayer/pull/3427#discussion_r290914876